### DISCUSSION TITLE
Logs are not getting piped up in discover

### DISCUSSION THREAD

**Discussion by bommasv** [Opster/opensearch-k8s-operator#818] [Q&A]
Until yesterday everything is good, all the logs are coming, up.

all of us sudden its not showing anything.

Expand your time range
One or more of the indices you’re looking at contains a date field. Your query may not match anything in the current time range, or there may not be any data at all in the currently selected time range. You can try changing the time range to one which contains data.

**Comment by swoehrl-mw** [comment]
Hi @bommasv. This sounds like its a problem with your specific opensearch setup/configuration and not something related to the operator. Please reach out to the general opensearch community to get help if needed.

**Reply by bommasv** [reply]
Thank You
